kuma/scrape/management/commands/sample_mdn.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from django.core.management.base import CommandError
+
+from kuma.scrape.fixture import FixtureLoader
+from . import ScrapeCommand
+
+
+class Command(ScrapeCommand):
+    help = 'Sample data from a running MDN server.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('spec',
```

Comment:
I had the same thought process. Default filenames with paths feels wrong as well, asking for Windows / Unix trouble. No default seemed the best course.